src/PhpParser/Node/Manipulator/ClassConstManipulator.php

```diff
@@ -29,27 +30,47 @@ final class ClassConstManipulator
      */
     private $betterStandardPrinter;
 
+    /**
+     * @var ParsedNodesByType
+     */
+    private $parsedNodesByType;
+
+    /**
+     * @var ClassManipulator
+     */
+    private $classManipulator;
+
     public function __construct(
         NameResolver $nameResolver,
         BetterNodeFinder $betterNodeFinder,
-        BetterStandardPrinter $betterStandardPrinter
+        BetterStandardPrinter $betterStandardPrinter,
+        ParsedNodesByType $parsedNodesByType,
+        ClassManipulator $classManipulator
     ) {
         $this->nameResolver = $nameResolver;
         $this->betterNodeFinder = $betterNodeFinder;
         $this->betterStandardPrinter = $betterStandardPrinter;
+        $this->parsedNodesByType = $parsedNodesByType;
+        $this->classManipulator = $classManipulator;
     }
 
     /**
      * @return ClassConst[]
      */
     public function getAllClassConstFetch(ClassConst $classConst): array
     {
+        /** @var Node\Stmt\Class_ $classNode */
```

Comment:
Here is `Class_|null` in reality.

I got many bugs because I assumed there have to be `Class_` :D